src/Stryker.CLI/README.md

```diff
@@ -68,9 +68,16 @@ If you want to decide on your own mutation score thresholds, you can configure t
 The defaults are respectivly: `80` `60` `0`
 
 #### Excluding files
-If any error occur in a specific source file you are able to exclude them from mutation. Execute the following command:
+If you decide to exclude files for unit testing, you can configure this with the following command:
 
-`dotnet stryker --files-to-exclude ['C:\\ExampleProject\\ExampleClass.cs', 'C:\\ExampleProject\\ExampleClass2.cs']
+`dotnet stryker --files-to-exclude ['./ExampleClass.cs','./ExampleDirectory/ExampleClass2.cs','C:\ExampleDirectory\ExampleClass.cs']`
+
+It is advisable to use relative paths. When someone else is running Stryker for you unit test project, paths are automatically resolved. It is still possible to use absolute paths to files.
+
+The following rules are used when using relative paths. For these rules the following example directory is used: C:\ExampleDirectory\OtherDirectory\
+
+1. When using the relative path option ./ (for Windows .\) it uses the current directory where it is scanning for files (The final directory is: C:\ExampleDirectory\OtherDirectory\ExampleClass.cs).
+2. When using the relative path option ../ (for Windows ..\) it moves up one directory that is being scanned (The final directory is: C:\ExampleDirectory\ExampleClass.cs).
```

Comment:
Well, I have added this due to the fact that most of the times (at least I have that problem :P) that when using relative paths that people don't know the difference. By this it gives more details of how to use the exclude files option :-). I can remove it though.

Reply:
Well, I'm now thinking that maybe we should handle paths platform unspecific. For example Stryker is configured for a team of 10 developers. Some might work on windows and others on mac. We would want Stryker to exclude the files on all platforms with the same config file. Could we achieve this? Maybe by replacing / to `Environment.PathSeperator` ?

This would also allow us to document this feature platform unspecific. That would be much nicer in my opinion!

Reply:
Finally I got a local windows ubuntu system running :-) so I can finally try to resolve this. I have discovered when using absolute paths C:\ExampleDirectory\Test.cs and when parsing it on ubuntu it gives a full path /someawesomedirectory/C:\ExampleDirectory\Test.cs which is ofcourse correct due to the fact that it tries to make use of the directory seperator where the tests are currently executed. With that said I think that absolute paths shouldn't be used and only accept relative paths. Otherwise it is not possible (yes everything is possible, but it needs some code which removes the filesystemroot) to create a config file that can be used by multiple environments/members. What do you think @richardwerkman?

Reply:
@richardwerkman I've updated the code. It now determines which kind of separator is being used and replaces it accordingly with the platform separator.